tests/bench_sizeof_coretypes/Makefile

```diff
@@ -0,0 +1,20 @@
+include ../Makefile.tests_common
+
+# Modules that will have an impact on the size of the TCB (thread_t):
+#
+# disabled by default, enable on demand:
+ifneq (,$(USE_FLAGS))
+  USEMODULE += core_thread_flags
+endif
+#
+# enabled by default, disable on demand:
+ifneq (,$(NO_MSG))
+  DISABLE_MODULE += core_msg
+endif
+
+TEST_ON_CI_WHITELIST += all
+
+include $(RIOTBASE)/Makefile.include
+
+test:
```

Comment:
Please remove the `test` target from your Makefile. It is now provided by `Makefile.include` (see [@cladmi's mail](https://lists.riot-os.org/pipermail/devel/2018-September/005918.html)).